{{cookiecutter.github_repo}}/scripts/build-run/test-entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# http://linuxcommand.org/lc3_man_pages/seth.html:
+# -e  Exit immediately if a command exits with a non-zero status.
+set -e
+
+echo Debug: $DEBUG
+
+# Collect static files
+echo "Collect static files"
+python manage.py collectstatic --noinput
+
+# have pytest run unit tests
+pytest
```

Comment:
I have some basic unit test cases to test that the endpoints resolve that I can add later.

Reply:
great, going to resolve for moment